ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -28,6 +28,12 @@ VAV:
   - EQUIPMENT
   opt_uses:
   - zone_use_label # Needed until the zones can are joined to assets with connections.
+  connections:
```

Comment:
this is not true for all equipment.

Reply:
Okay, assuming this is in reference to `zone_use_label`, deleting